backend/constants.js

```diff
@@ -476,6 +476,15 @@
 	define("USERNAME_REGEXP",  /^[a-zA-Z][\w]{1,19}$/);
 	define("EMAIL_REGEXP", /^([a-zA-Z0-9_\-.]+)@([a-zA-Z0-9_\-.]+)\.([a-zA-Z]{2,5})$/);
 
+	// -----------------------------------------------------------------------------
+	//
+	// Password requirements
+	//
+	// -----------------------------------------------------------------------------
+
+	define("MIN_PASSWORD_LENGTH", 8);
```

Comment:
Yes, I branched this off the password one to minimise conflicts as I thought it would be touching the same files. Could we merge the password one in first when it's ready?